modules/ipi-install-configuring-networking.adoc

```diff
@@ -28,87 +32,147 @@ $ export PUB_CONN=<baremetal_nic_name>
 +
 [NOTE]
 ====
-The SSH connection might disconnect after executing these steps.
+When configuring the bare-metal network and the SSH connection disconnects, NMState has a rollback mechanism that automatically reverts any configurations. You can also `nmstatectl gc` tool to view garbage collected configuration snapshots.
 ====
-
-.. For a network using DHCP, run the following command:
++
+.. For a network that uses Dynamic Host Configuration Protocol (DHCP), create an NMState YAML file and specify the bare-metal bridge interface and any physical interfaces in the file:
++
+.Example bare-metal bridge interface configuration that uses DHCP
+[source,yaml]
+----
+# ...
+interfaces:
+  - name: baremetal
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port:
+        - name: <physical_interface_name> <1>
+  - name: <physical_interface_name>
+    type: ethernet
+    state: up
```

Comment:
We have to define the physical interface first, and clear the IP address from it.

NMstate defines everything at once, so when we create the bridge we must also configure the base interfaces.